test/test_transforms_v2_refactored.py

```diff
@@ -2437,3 +2437,17 @@ def test_correctness(self):
                 assert isinstance(out_value, torch.Tensor) and not isinstance(out_value, datapoints.Datapoint)
             else:
                 assert isinstance(out_value, type(input_value))
+
+
+class TestGaussianBlur:
+    @pytest.mark.parametrize(
+        "make_input",
+        [make_image_tensor, make_image_pil, make_image, make_bounding_boxes, make_segmentation_mask, make_video],
+    )
+    @pytest.mark.parametrize("device", cpu_and_cuda())
+    def test_transform(self, make_input, device):
+        check_transform(transforms.GaussianBlur, make_input(device=device), kernel_size=3, sigma=(0.5, 2.0))
+
+    def test_input_args(self):
+        t = transforms.GaussianBlur(3, sigma=(0.5, 2))
+        assert t.sigma == [0.5, 2.0]
```

Comment:
Could we drop the instance attribute check here? The fact that we internally store `sigma` as list of floats is an implementation detail

```suggestion
    def test_sigma_int(self):
        # Non-regression test for https://github.com/pytorch/vision/issues/7885
        transforms.GaussianBlur(kernel_size=3, sigma=(0.5, 2))
```

We can likely also put a note on top of the test to remove it when we have ported it, because such a check will likely be included in the "regular" tests as well.